crawl-ref/source/throw.cc

```diff
@@ -854,6 +854,13 @@ bool throw_it(bolt &pbolt, int throw_2, dist *target)
                     monster *am = monster_at(*ai);
                     if (am)
                         cancelled = stop_attack_prompt(am, false, *ai);
+                    else if (*ai == you.pos())
+                    {
+                        string prompt = make_stringf("That %s is likely to hit you. Continue anyway?",
```

Comment:
Maybe just "That is likely to hit you. Continue anyway?" is sufficient?

Reply:
Sure, that works. Updated.